### ISSUE TITLE
Cannot read property 'some' of undefined (while running with babel-node/babel-cli)

### ISSUE THREAD

**Issue by philipodev** [kentcdodds/babel-plugin-preval#5]
- `babel-plugin-preval` version: 1.2.0
- `node` version: 6.7.0
- `yarn` version: 0.23.2

Relevant code or config

`package.json` scripts
```json
{
    "scripts": {
        "start": "npm-run-all --parallel test:watch open:src lint:watch",
        "open:src": "babel-node tools/srcServer.js", // note this (babel-node)
    }
}
```

`.babelrc`
```json
{
    "presets": ["react", "stage-1"],
    "env": {
        "development": {
            "plugins": ["preval"],
            "presets": ["env", "react-hmre"]
        },
        "production": {
            "presets": [
                [
                    "env",
                    {
                        "es5": {
                            "modules": false
                        }
                    }
                ]
            ],
            "plugins": [
                "transform-react-constant-elements",
                "transform-react-remove-prop-types"
            ]
        },
        "test": {
            "presets": ["env"]
        }
    }
}
```
`tools/srcServer`:
[https://github.com/coryhouse/react-slingshot/blob/770e88e2bd32067acb59cf3c62e0f051f3621a42/tools/startMessage.js](https://github.com/coryhouse/react-slingshot/blob/770e88e2bd32067acb59cf3c62e0f051f3621a42/tools/startMessage.js)

What you did:
`yarn start`
*Running babel-node (babel-cli) on a node-script with `preval` as plugin in `.babelrc` throws an error.*

What happened:

```terminal
C:\Users\PA\Desktop\react-playground\preval-issue (master) (test@0.1.0)
λ yarn start
yarn start v0.23.2
$ npm-run-all --parallel start-message

> test@0.1.0 start-message C:\Users\PA\Desktop\react-playground\preval-issue
> babel-node tools/startMessage.js

C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-register\node_modules\babel-core\lib\transformation\file\index.js:590
      throw err;
      ^

TypeError: C:/Users/PA/Desktop/react-playground/preval-issue/tools/startMessage.js: Cannot read property 'some' of undefined
    at leadingComments (C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:66:32)
    at C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:193:14
    at Array.every (native)
    at looksLike (C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:189:35)
    at C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:195:48
    at Array.every (native)
    at looksLike (C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:189:35)
    at C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:195:48
    at Array.every (native)
    at looksLike (C:\Users\PA\Desktop\react-playground\preval-issue\node_modules\babel-plugin-preval\dist\index.js:189:35)
```

Reproduction repository:
[coryhouse/react-slingshot](https://github.com/coryhouse/react-slingshot/)
* clone
* `yarn run setup` (takes about 2 min)
* `yarn add --dev babel-plugin-preval`
* add `"preval"` to `babel.env.development.plugins[]` in `package.json`
* `yarn start`

Problem description:
argument `comments` is `undefined` here: [/src/index.js#L62](https://github.com/kentcdodds/babel-plugin-preval/blob/master/src/index.js#L62) but still tries to call function `some`.

Suggested solution:
make sure `comments` is not `undefined` before calling property `some`.
But if `comments` is `undefined` what should we return? `false`?

**Comment by philipodev** [comment]
i would love to submit an PR on this issue but i'm unsure what to return if `comments` is `undefined`.

**Comment by kentcdodds** [comment]
Thanks a ton for the thorough bug report! Just exit early if there are no comments. That will be fine. Please include a test. Thanks a bunch!

**Comment by kentcdodds** [comment]
This was fixed in #6, thanks for the bug report @philipodev!
